scripts/fuzz_opt.py

```diff
@@ -1749,6 +1748,40 @@ def can_run_on_wasm(self, wasm):
         return not CLOSED_WORLD and all_disallowed(['shared-everything']) and not NANS
 
 
+# Test --fuzz-preserve-imports-exports, which never modifies imports or exports.
+class PreserveImportsExports(TestCaseHandler):
+    frequency = 0.1
+
+    def handle(self, wasm):
+        # Generate some random input data.
+        data = abspath('preserve_input.dat')
+        make_random_input(random_size(), data)
+
+        # Process the existing wasm file.
+        processed = run([in_bin('wasm-opt'), data] + FEATURE_OPTS + [
+            '-ttf',
+            '--fuzz-preserve-imports-exports',
+            '--initial-fuzz=' + wasm,
+            '--print',
+        ])
+
+        # Verify no imports or exports changed, by comparing to unprocessed/
+        # original text.
+        original = run([in_bin('wasm-opt'), wasm] + FEATURE_OPTS + ['--print'])
+
+        def get_relevant_lines(wat):
+            # Imports and exports are relevant.
+            lines = [line for line in wat.splitlines() if '(export ' in line or '(import ' in line]
+
+            # Ignore type names, which may vary from $5 to $17 in uninteresting
```

Comment:
Is it `$5 to $17` because of the particular contents of `preserve_input.dat`? This comment seems very likely to become stale at some point. Can we generalize it?

Reply:
Ah, that is just an example. I'll clarified it now.